source/appModules/notepad.py

```diff
@@ -0,0 +1,31 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Joseph Lee
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""App module for Windows Notepad.
+While this app module also covers older Notepad releases,
+this module provides workarounds for Windows 11 Notepad."""
+
+import appModuleHandler
+import api
+
+
+class AppModule(appModuleHandler.AppModule):
+
+	def _get_statusBar(self):
+		# #13688: Notepad 11 uses Windows 11 user interface, therefore status bar is harder to obtain.
+		# This does not affect earlier versions.
+		notepadVersion = int(self.productVersion.split(".")[0])
+		if notepadVersion < 11:
+			raise NotImplementedError()
+		# And no, status bar is shown when editing documents.
+		# Thankfully, of all the UIA objects encountered, document window has a unique window class name.
+		if api.getFocusObject().windowClassName != "RichEditD2DPT":
+			raise NotImplementedError()
+		# Look for a specific child as some children report the same UIA properties such as class name.
+		statusBar = api.getForegroundObject().children[7].firstChild
```

Comment:
Could you add a brief note on how you determined that this is the index? Is there any MS documentation for this?

```suggestion
		# Look for a specific child as some children report the same UIA properties such as class name.
		notepadStatusBarIndex = 7
		statusBar = api.getForegroundObject().children[notepadStatusBarIndex].firstChild
```